stacks/roundabout-stack.js

```diff
@@ -34,6 +34,8 @@ export function RoundaboutStack({ stack, app }) {
     routes: {
       'GET /{cid}':       'functions/redirect.handler',
       'HEAD /{cid}':      'functions/redirect.handler',
+      'GET /raw/{key}':   'functions/redirect.rawHandler',
+      'HEAD /raw/{key}':   'functions/redirect.rawHandler',
```

Comment:
Key has to be with URL encoded slashes right or this will not work...?

Reply:
dagcargo has no `/` so it was fine. But if we do a single endpoint we need to take that into account yes